src/helpers.ts

```diff
@@ -39,39 +40,40 @@ export function isSimplePath(maybePath: string, ctxt: object): boolean {
   return ctxt ? Object.prototype.hasOwnProperty.call(ctxt, path) : false;
 }
 
-type Args = string | object | object[];
+type Args = string | UnknownObject | UnknownObject[];
 
 /**
  * Recursively apply a template renderer to a configuration.
  */
 export function mapArgs(
   config: string,
-  ctxt: object,
+  ctxt: UnknownObject,
   render?: Renderer
 ): string;
+// eslint-disable-next-line @typescript-eslint/ban-types -- TODO: Type too broad
 export function mapArgs<T extends object>(
   config: T,
-  ctxt: object,
+  ctxt: UnknownObject,
   render?: Renderer
 ): T;
 export function mapArgs(
-  config: object[],
-  ctxt: object,
+  config: UnknownObject[],
+  ctxt: UnknownObject,
   render?: Renderer
-): object[];
+): UnknownObject[];
 export function mapArgs(
   config: Args,
-  ctxt: object,
+  ctxt: UnknownObject,
   render: Renderer = Mustache.render
 ): unknown {
   if (Array.isArray(config)) {
     return config.map((x) => mapArgs(x, ctxt, render));
   }
 
-  if (isPlainObject(config)) {
+  if (isPlainObject(config) && typeof config === "object") {
```

Comment:
Why is the type check needed if there's a isPlainObject call? Because the isPlainObject doesn't do a type guard?

Reply:
> Because the isPlainObject doesn't do a type guard

Yes, unfortunately. I could replace it with this though
```suggestion
  if (config && typeof config === "object") {
```